woocommerce/woocommerce-android
# [Simplified login] UI for Jetpack Activation steps

Part of: #7848 


⚠️ Please don't merge before #7872 

### Description
This PR adds the UI part for the installation steps.

### Testing instructions
Either launch the preview `JetpackActivationMainScreenPreview` or follow the below steps
1. Open the site picker (from login or settings)
2. Click on "Add a store"
3. Click on "Connect an existing store"
4. Enter the address of a non-jetpack store.
5. Click on "Install Jetpack".
6. Enter site credentials and click on "Install Jetpack"
7. Confirm the installation steps are shown like the below screenshots.
8. Click on the Close button.
9. Confirm you are navigated to the initial screen.

### Images/gif
| Light | Dark |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1657201/202426585-1bee5a54-8d86-460f-b95f-ee8c1fca9e0e.png) | ![image](https://user-images.githubusercontent.com/1657201/202426697-7a878b77-9b20-4d4b-a82d-bdf3ed6793f0.png) |
